index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./normalize.css"/>
+    <link rel="stylesheet" type="text/css" href="./style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
+    <script src="./script.js" defer></script> 
+    <title>Surfing Penguins - JS Game. </title>
+</head>
+<body>
+    <main>
+        <div class="landingTopContainer">
+            <div class="landingLeft"></div>
+            <div class="landingTitleContainer">
+                <h1 class="landingTitle">SURFWAVE Fury</h1>
+            </div>
+            <div class="leaderSurfboardContainer">
+                <button id="leaderSurfboard">Leader Surf Board</button>
+            </div>
+        </div>
+        <div class="mainContainer">
+            <div id="mainGameContainer">
+                <div class="cloud-background"></div>
+                <div class="sun"></div>
+                <div id="avatar"></div>
+                <div class="ocean">
+                  <div class="wave"></div>
+                  <div class="wave"></div>
+                  <div class="wave"></div>
+                </div>
+            <div class="waves"></div>
+            <div id="displaySplashInstructionsContainer">
+                <h2 class="instructionsOverlayTitle">How to play the game?</h2>
+                <div class="intructionsOverlayText">
+                    <p>Aloha!</p>
+                    <p>Get ready to ride the wave dude!</p>
+                    <p>Just click to start, then when you're ready hit the LEFT &#8592 then RIGHT &#8594 buttons alternately, bro!
+                    Go as fast as you can but make sure you don't press the same button twice in a row because you'll be stuck in the foam!</p>
+                    <p>Shakas brah! &#129305</p>
```

Comment:
You should end the character references with a ;

for example `&#129305;`

Reply:
Amended this one and all of the same character references.